### PR TITLE
Require cloud build to succeed in make dist checks.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

This updates our `make dist` check to only pass if building with Netdata Cloud functionality succeeds.

##### Component Name

area/ci

##### Test Plan

CI check passes on this PR.

##### Additional Information

Fixes: #8284

### CODE REVIEW

**Comment by Ferroin** [comment]
> Should we refactor `.github/scripts/run_install_with_dist_file.sh` to accept arguments we can pass in from CI and have a few different things we can check? Not just assume that all checks will now ensure the Cloud functionality builds correctly, but other optional features as separate jobs?

I'd been thinking about this myself, but a lot more generically than just that.

**Comment by Ferroin** [comment]
@prologic Could you possibly take a look into the failures here, I'm stumped. It looks like for some reason `install-required-packages.sh netdata` isn't pulling in `cmake`, but only in this particular GHA instance (if it weren't anywhere else, other checks would be failing and users would be complaining).

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Prepare environment
         run: |
-          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata-all
+          ./packaging/installer/install-required-packages.sh --dont-wait --non-interactive netdata
```
Comment (prologic) [inline .github/workflows/checks.yml:64]: I plan to remove `netdata` as an option to this tool.

**Comment by prologic** [comment]
@Ferroin You need to see what the dist build failures are. The CentOS 8.x failures are a KP and I'm working on them. Please have a look :)

File: .github/workflows/checks.yml (hunk 1 above)
Comment (Ferroin) [reply .github/workflows/checks.yml:64]: Unless you're changing 'netdat-all' to be behave like the current 'netdata' option does, don't. We should not be unconditionally pulling in Node.js Python, and all of our Python dependencies on installs on user systems.

**Comment by Ferroin** [comment]
@prologic Finally got around to updating this and getting it working correctly, if you could take another look at it that would be appreciated.

**Comment by prologic** [comment]
Merged on behalf of @Ferroin (_while away on vacation_); as this directly impacts our CI and improves it.

**Pull request merged**